### PR TITLE
Added context menu option in Bundles to add items directly to a Bundle

### LINKED ISSUE #5256: Improve UX for adding new items to bundles
**Is your feature request related to a problem?**
Once a bundle contains items, the tooltip explaining how to add more to it goes away. If a user set up their bundles and then forgot, there is no way to figure out how to add new items without creating a new bundle.

**Describe the solution**
Adding an option to add a new folder to a bundle via the right click menu.

**Describe alternatives you've considered**
Adding an option to add a new folder to a bundle via the 3-dot menu.

**Scope**

| Capability | Priority |
| :---------- | :------- |
| This proposal will add the button to the right click menu | Must |
| This proposal will add the button to both the right click and 3-dot menu | Could |
| This proposal replace the old drag-drop system. | Won't |

**System Information**
- Device name: TheOnlyTails
- Processor: Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz   2.30 GHz
- Installed RAM: 8.00 GB (7.85 GB usable)
- Device ID: EFEC9023-BBDC-4447-ADCD-269D64BDF32C
- Product ID: 00325-96606-60041-AAOEM
- System type:	64-bit operating system, x64-based processor
- Pen and touch: No pen or touch input is available for this display
- App version: v1.5.44.0

### PR DESCRIPTION
**Pull request by d2dyno1**
**Resolved / Related Issues**
- Closes #5256

**Details of Changes**
This PR adds a context menu option in bundles to directly select and add file/folder to a bundle. This PR also improves performance by utilizing `{x:Bind}` and `x:Load` replacing `{Binding}` and `Visibility`, respectively. Further improvements have been made to code as well, featuring use of `AsyncRelayCommand` and `Task.WaitAll()` for better stability.

**Validation**
How did you test these changes?
- [x] Built and ran the app

**Screenshots (optional)**
image

image

### CODE REVIEW

File: Files/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -2622,4 +2622,13 @@
   <data name="NavigationToolbarCopy.Label" xml:space="preserve">
     <value>Copy</value>
   </data>
+  <data name="BundleOptionsFlyoutAddFileMenuItem" xml:space="preserve">
```
Comment (gave92) [inline Files/Strings/en-US/Resources.resw:2625]: Shouldn't this be "BundleOptionsFlyoutAddFileMenuItem.Text"?

**Comment by gave92** [comment]
Looks like items added though the new "Add item" menu are not persisted across app restart.

File: Files/Strings/en-US/Resources.resw (hunk 1 above)
Comment (d2dyno1) [reply Files/Strings/en-US/Resources.resw]: Thanks! Fixed in https://github.com/files-community/Files/pull/5475/commits/88ff9c11d4a61063733cbbc1ca5f47b9c2d1454c

**Review by gave92 — APPROVED** [review]

**Comment by yaichenbaum** [comment]
Thank you!

**Review by phyllybb — COMMENTED** [review]
Yes

**Pull request merged**